### ISSUE TITLE
value_from doesn't support long double

### ISSUE THREAD

**Issue by pdimov** [boostorg/json#635]
n/t

**Comment by grisumbras** [comment]
Do you think we also need a constructor from `long double`?

**Comment by pdimov** [comment]
No, I don't think we do. Constructors are only for "union members" of `value`, whereas `value_from` can take anything.

**Comment by grisumbras** [comment]
For better or worse, we have `value` constructors from every standard integer type.
